src/python/pants/rules/core/python_test_runner.py

```diff
@@ -0,0 +1,31 @@
+# coding=utf-8
+# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from builtins import str
+
+from pants.engine.legacy.graph import HydratedTarget
+from pants.engine.rules import rule
+from pants.engine.selectors import Select
+from pants.rules.core.core_test_model import GenericTestResult, Status
+
+
```

Comment:
So, I think I just remembered that #6652 isn't quite accurate: commented there.

You should be able to declare this in `src/python/pants/backend/python/register.py`.

Reply:
Will move tomorrow :)